sdk/elastic/Azure.ResourceManager.Elastic/src/autorest.md

```diff
@@ -53,4 +53,10 @@ acronym-mapping:
 rename-mapping:
   MonitoredResource: MonitoredResourceContent
 
+directive:
+  - from: elastic.json
+    where: $.definitions
+    transform: >
+      $.ElasticTrafficFilter.properties.type['x-ms-client-name']= 'ElasticTrafficFilterType'
```

Comment:
we usually do not use directive to rename things, could you remove this and use our `rename-mapping` instead?
This should be something like:
```
rename-mapping:
	ElasticTrafficFilter.type: ElasticTrafficFilterType
```

Reply:
Done